internal_public.go

```diff
@@ -148,27 +162,27 @@ func NewActivityTaskHandler(service m.TChanWorkflowService, identity string, log
 func AddWorkflowRegistrationInterceptor(
```

Comment:
Should these workflow/activity interceptors be moved to per worker? instead of global, since they are bound to registration scope.